### ISSUE TITLE
Rancher CLI: rm host fails with Bad response statusCode [405]

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#6376]
Rancher Version:v1.2.0-pre4-rc4

Rancher CLI version: v0.3.0-rc3

OS and where are the hosts located? (cloud, bare metal, etc):: Cloud

Environment Type: (Cattle/Kubernetes/Swarm/Mesos): Cattle

Steps to Reproduce:
1. Create a rancher cli setup and use account API keys
2. Run the command ./rancher rm --type host 1h2

```
soumyas-MacBook-Pro:rancher-v0.3.0-rc3 soumya$ ./rancher rm --type host 1h2
error 1h2: Bad response statusCode [405]. Status [405 Method Not Allowed]. Body: [] from [http://104.196.211.99:8080/v2-beta/projects/1a5/hosts/1h2]
1h2
```

**Comment by deniseschannon** [comment]
Able to remove hosts using `./rancher rm --type host 1h5`

Verified fixed with Rancher CLI v0.4.0-rc1, and rancher in v1.2.0-rc2.

**Comment by bacheson** [comment]
seeing this exact issue in 1.3.2
